dotcom-rendering/src/web/components/SignInGateSelector.importable.tsx

```
@@ -102,7 +102,7 @@ const ShowSignInGate = ({
 	useEffect(() => {
 		submitViewEventTracking({
 			component: withComponentId(
-				signInGateTestIdToComponentId[abTest.id],
+				signInGateTestIdToComponentId[abTest.id] ?? 'unknown',
```

@guardian/identity you may want to have a peek.

---

just curious about the choice of `unknown` as a default here — is this a standard thing?

---

Not particularly, hence the nudge.

I'm not sure what would make the most sense here, maybe you want to throw an error?